star-ros2/src/star_perception/star_perception/hailo_runner.py

```
@@ -0,0 +1,489 @@
+    decoded: list[tuple[int, float, float, float, float, float]] = []
+    if not outputs:
+        return decoded
+    name = next(iter(outputs))
+    arr = outputs[name]
+    if arr.ndim == 4 and arr.shape[0] == 1:
+        arr = arr[0]
+    # arr shape now: (num_classes, max_dets, 5) or list-of-arrays.
+    if isinstance(arr, list):
+        per_class = arr
+    else:
+        per_class = list(arr)
```

Ruff (0.15.10)

[warning] Use ternary operator `per_class = arr if isinstance(arr, list) else list(arr)` instead of `if`-`else`-block

Replace `if`-`else`-block with `per_class = arr if isinstance(arr, list) else list(arr)`

(SIM108)